### ISSUE TITLE
Ordered lists not rendered correctly

### ISSUE THREAD

**Issue by BijoySingh** [noties/Markwon#21]
I have been using Markwon in one of my apps, and I have this issue which I have been trying to solve. 
https://github.com/BijoySingh/Material-Notes-Android-App/issues/22

@noties Are you aware why something like this could happen?

**Comment by noties** [comment]
Hey! Can you point me to the layout file that contains TextView that displays markdown in your app?

**Comment by BijoySingh** [comment]
The layout file: https://github.com/BijoySingh/Material-Notes-Android-App/blob/master/app/src/main/res/layout/item_format_text.xml

This is the textview:
```
  <TextView
    android:id="@+id/text"
    style="@style/FormatText"
    android:autoLink="web"
    android:hint="@string/format_hint_text"
    android:linksClickable="true"
    android:textSize="@dimen/text_font_size" />
```

This is the style:
```
  <style name="FormatText">
    <item name="android:layout_width">0dp</item>
    <item name="android:layout_weight">1</item>
    <item name="android:layout_height">wrap_content</item>
    <item name="android:gravity">top</item>
    <item name="android:layout_gravity">center</item>
    <item name="android:background">@color/transparent</item>
    <item name="android:fontFamily">monospace</item>
    <item name="android:lineSpacingExtra">@dimen/extra_text_line_spacing</item>
    <item name="android:padding">8dp</item>
    <item name="android:textColor">@color/dark_secondary_text</item>
    <item name="android:textColorHint">@color/dark_hint_text</item>
    <item name="android:textSize">16sp</item>
</style>
```

**Comment by noties** [comment]
Fixed in `v1.0.3`
